test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+require 'pry'
+
+describe RecipesController do
+
+  describe 'index' do
+    it "should get index path" do
+      VCR.use_cassette("recipes") do
+        get recipes_path
+      end
+
+      must_respond_with :ok
+    end
+
+  end
+
+  describe 'show' do
```

Comment:
Missing tests for the `show` action.

Something like
```ruby
it "will show a valid recipe" do
  VCR.use_cassette("recipes") do
    get recipe_path("http%3A%2F%2Fwww.edamam.com%2Fontologies%2Fedamam.owl%23recipe_95e8e9135376ecf57440f594d2e0c530")

    must_respond_with :ok
  end
end
```

Then create a test with an invalid URI.